Sources/Slacket/Parsers/ParserType.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ParserType.swift
+//  Slacket
+//
+//  Created by Jakub Tomanik on 20/05/16.
+//
+//
+
+import Foundation
+import Kitura
+
+// To define models that can be parsed
+protocol Parsable {}
+
+// To define parsers
+protocol ParserType {}
+
+protocol ParserDecoder: ParserType {
+    
+    func parse(body: ParsedBody) -> Parsable
```

Comment:
Keep in mind that if
`struct SlackCommand: Parsable {`
then

``class SlackCommandDecoder: ParserDecoder {
func parse(body: ParsedBody) -> SlackCommand {
}
}``

and we're both right :)